src/main/kotlin/com/openlattice/linking/LinkingFeedbackApi.kt

```diff
@@ -50,7 +50,7 @@ interface LinkingFeedbackApi {
      * Returns positive/negative/all linking feedbacks on the given entity.
```

Comment:
comments could be updated too

Reply:
+ `LinkingFeedbackController` should be updated too
+ rehearsal tests, that call these function too

Reply:
Ehhh this was clearly too much effort for what it's worth, but here it goes :) 

https://github.com/openlattice/rehearsal/pull/48
https://github.com/openlattice/conductor-client/pull/627
https://github.com/openlattice/linker/pull/29